<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewItemWidget</class>
 <widget class="QWidget" name="ReviewItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-family: Open Sans;
	font-size: 18px;
	font-weight: 400;
	line-height: 24px;
	letter-spacing: 0%;
	color: #1A1A1A;
}

#endVerValueLabel,
#startVerValueLabel {
	font-weight: 600;
}

QPushButton {
	background: transparent;
	border: none;
	font-family: Open Sans;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	color: #1A1A1A;
	text-decoration: underline;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="mainLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="startVerTextLabel">
     <property name="text">
      <string>Start Version:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="startVerValueLabel">
     <property name="text">
      <string>2.3.4.5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="endVerTextLabel">
     <property name="text">
      <string>End Version:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="endVerValueLabel">
     <property name="text">
      <string>2.3.4.6</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="showFindingButton">
     <property name="text">
      <string>Show Findings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="viewDetailButton">
     <property name="text">
      <string>View Details</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
